docs/howto/retransform_datasets.md

```diff
@@ -0,0 +1,29 @@
+# How to re-transform a Dataset
+
+ Re-transforming a dataset is necessary when any of the following tasks are completed in Horreum
```

Comment:
I still think this is rather succinct on the *why*s - it is not *necessary* to re-transform the when any of the below changes. You might be perfectly fine with keeping the existing datasets the way they are, and Horreum won't blow up if you don't re-transform.
On the other hand I find the list below rather verbose (to last 4 points could be summarized as changing the Transformer definition in Schema), but you can list them all if you think it contributes to clarity.

Reply:
The _whys_ have been summarized and expanded.